### PR TITLE
[Php70] Skip class has __callStatic() on StaticCallOnNonStaticToInstanceCallRector

### PR DESCRIPTION
**Pull request by samsonasik**
It can happen to call extended or other class, it should be skipped ref https://3v4l.org/6oHBd 
Without static call, it will result method not found, ref https://3v4l.org/DXCHc

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules-tests/Php70/Rector/StaticCall/StaticCallOnNonStaticToInstanceCallRector/Fixture/skip_callstatic.php.inc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    }
+}
+
+class SkipCallStatic
+{
+    public function run()
+    {
+        return Service::getAlbum();
+    }
+}
```
Comment (TomasVotruba) [inline rules-tests/Php70/Rector/StaticCall/StaticCallOnNonStaticToInstanceCallRector/Fixture/skip_callstatic.php.inc:13]: It's a bit unclear now what class is being refactored and which classes are here only to make the analysis work.
Could you move "source" classes that are not being refactored to the `/Source` directory?

Comment (samsonasik) [reply rules-tests/Php70/Rector/StaticCall/StaticCallOnNonStaticToInstanceCallRector/Fixture/skip_callstatic.php.inc:13]: implemented, moved to `/Source` https://github.com/rectorphp/rector-src/pull/555/commits/cfaee2464de39527577be1e515ac5443bac8c233

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

File: rules-tests/Php70/Rector/StaticCall/StaticCallOnNonStaticToInstanceCallRector/Fixture/skip_callstatic.php.inc (hunk 1 above)
Comment (TomasVotruba) [reply rules-tests/Php70/Rector/StaticCall/StaticCallOnNonStaticToInstanceCallRector/Fixture/skip_callstatic.php.inc:13]: Thank you :+1:

**Pull request merged**